README.md

```diff
@@ -4,6 +4,7 @@ This repository offers utilities for Bitcoin and RGB++ asset integration.
 
 ### Packages in this repository
 
+- [rgbpp](./packages/rgbpp): A root package to integrate of common functions from the RGB++ SDK sub-packages(btc/ckb/service).
```

Comment:
### A Thought
What about revealing this root package in `/README` after some testing, after the code examples is updated?

Reply:
Make sense